### ISSUE TITLE
Problem with authentication on 2nd device

### ISSUE THREAD

**Issue by iTommix** [nodogsplash/nodogsplash#602]
I have NDS running with FAS and bin_auth on openWRT.

If i do a fresh start of the system i experience this:

The first device that binds to the wifi will get the splash page to login. After login that device got access. The 2nd device can bind and dont have to login and got access immediately.

I tested this with an iPhone and Android. I tried vice versa and faced this behavior on both devices.

Any ideas?

Thomas

**Comment by bluewavenet** [comment]
@iTommix 
NodogSplash no longer supports FAS/BinAuth.
Please upgrade to openNDS and see if the problem is fixed. 
If not or if you have any other questions please open an issue on openNDS:
https://github.com/openNDS/openNDS

**Comment by iTommix** [comment]
@bluewavenet 

Yes... i've read about. But because it partly worked i guessed it's still alive in NDS.

Will move to openNDS.

Thanks
